opentech/apply/projects/models.py

```diff
@@ -11,10 +12,16 @@ class Project(models.Model):
 
     title = models.TextField()
 
-    contact_legal_name = models.TextField(default='')
-    contact_email = models.TextField(default='')
-    contact_address = models.TextField(default='')
+    contact_legal_name = models.TextField(_('Person or Organisation name'), default='')
+    contact_email = models.TextField(_('Email'), default='')
+    contact_address = models.TextField(_('Address'), default='')
+    contact_phone = models.TextField(_('Phone'), default='')
     value = models.DecimalField(default=0, max_digits=10, decimal_places=2)
```

Comment:
Do we want a min number validator on this? Should have been on the model definition PR but was thinking about it when reviewing the form.

`validators=[MinValueValidator(Decimal('0.01'))`

https://docs.djangoproject.com/en/dev/ref/validators/#minvaluevalidator

Reply:
Fixed in dc518330